tv-shows-app/src/components/features/shows/ShowReviewSection/ShowReviewSection.tsx

```diff
@@ -0,0 +1,58 @@
+'use client';
+import { IReview } from "@/typings/Review.type";
+import { ReviewList } from "../../review/ReviewList/ReviewList";
+import { useEffect, useState } from "react";
+import { ReviewForm } from "../ReviewForm/ReviewForm";
+import { Flex } from "@chakra-ui/react";
+
+interface IShowReviewSectionProps{
+    updateRating: (rating:number) => void;
+}
+
+export const ShowReviewSection = ({updateRating}:IShowReviewSectionProps) =>{
+
+    const [reviews, setReviews] = useState([] as IReview[]);
+
+    const loadReviewsFromLocalStorage = () => {
+        const reviewsString = localStorage.getItem('reviews');
+        if(!reviewsString){
+            return [];
+        }
+        return JSON.parse(reviewsString) as IReview[];
+    }
+    
+    useEffect(()=>{
+        setReviews(loadReviewsFromLocalStorage);
```

Comment:
```suggestion
        const loadedReviews = loadReviewsFromLocalStorage();
        setReviews(loadedReviews);
        updateRating(loadedReviews.reduce((sum, r)=> r.rating + sum, 0)/loadedReviews.length);
```

Zašto nisam napravio ovako nešto?
```
        setReviews(loadReviewsFromLocalStorage());
        updateRating(reviews.reduce((sum, r)=> r.rating + sum, 0)/reviews.length);
```